apistrap/types.py

```diff
@@ -8,7 +9,7 @@
 
 @dataclass
 class FileResponse:
-    filename_or_fp: Union[str, bytes]
+    filename_or_fp: Union[str, bytes, BytesIO]
```

Comment:
Any file-like object should work here. Isn't there a type hint for that?

Reply:
OK, merging with `typing.io.BinaryIO`, see https://stackoverflow.com/questions/38569401/type-hint-for-a-file-or-file-like-object